games-simulation/openrct2/openrct2-0.0.7.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.website/"
+MY_REPO="https://github.com/OpenRCT2/OpenRCT2"
+if [ "${PV}" == "9999" ]; then
+	EGIT_REPO_URI="${MY_REPO}.git"
+	EGIT_BRANCH="develop"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="${MY_REPO}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/OpenRCT2-${PV}"
+fi
+SRC_URI+="
+https://github.com/OpenRCT2/title-sequences/releases/download/v0.0.5/title-sequence-v0.0.5.zip -> ${PN}-title-sequence-v0.0.5.zip
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+twitch +multiplayer opengl truetype libressl test"
+
+# This is needed because of this bug: https://github.com/OpenRCT2/OpenRCT2/issues/5469
+REQUIRED_USE="multiplayer? ( twitch )"
+
+RDEPEND="
+	twitch? ( net-misc/curl[ssl] )
+	multiplayer? (
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
+	opengl? ( virtual/opengl )
+	truetype? (
+		media-libs/sdl2-ttf
+		media-libs/fontconfig
+	)
+	media-libs/libsdl2
+	|| (
+		(
+			>=media-libs/speex-1.2.0
+			media-libs/speexdsp
```

Comment:
I prefer keeping it mainly because with >=speex-1.2.0 a [workaround](https://github.com/lorem-ipsum/ebuilds/blob/master/games-strategy/openrct2/openrct2-0.0.7.ebuild#L65-L66) is required (which I'm including) and that indicates it is not perfectly compatible (it works just fine with the stable speex). Also since speex is not stable __right now__, so why not?